crates/gpui/src/platform/windows/events.rs

```diff
@@ -659,6 +659,13 @@ fn handle_calc_client_size(
     requested_client_rect[0].left += frame_x + padding;
     requested_client_rect[0].bottom -= frame_y + padding;
 
+    if state_ptr.state.borrow().is_maximized() {
+        requested_client_rect[0].top += frame_y + padding;
+    } else {
+        // Extra pixel from the top border
```

Comment:
only thing I'd maybe change is add a reason for the extra pixel

also is this 1 pixel DPI dependent? maybe mention that. Either way I'm good with it from this point on. I'll leave it to the Zed team from here on :)

Reply:
I'm actually not sure if the thin 1 pixel border is DPI dependent, I would assume no but I don't actually know.

Reply:
I don't know how to get the border width that works with scaling, I tried `SM_CXBORDER` but it is always reporting 1 pixel for me.